### DISCUSSION TITLE
a/try-google-gemma-7b-with-ollama

### DISCUSSION THREAD

**Discussion by giscus** [sincerefly/giscus-data#10] [Announcements]
# a/try-google-gemma-7b-with-ollama

https://blog.yasking.org/a/try-google-gemma-7b-with-ollama.html

**Comment by kangfenmao** [comment]
给你推荐一个我开发的 Cherry Studio 客户端，支持 Ollama: https://github.com/kangfenmao/cherry-studio

**Reply by sincerefly** [reply]
有机会试试 👍
